src/index.html

```diff
@@ -6,9 +6,24 @@
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Moyo header</title>
+  <link href="https://fonts.googleapis.com/css?family=Roboto:400,500&display=swap&subset=cyrillic" rel="stylesheet">
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>Moyo header</h1>
+  <header class="header">
+    <a class="logo" href="/"><img src="images/logo.png" alt="logo" width="40px" height="40px"></a>
```

Comment:
don't write inline styles
```suggestion
    <a class="logo" href="/"><img src="images/logo.png" alt="logo"></a>
```

Reply:
Ok, that's I will remember too